CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [3.17.11.0]
+### Fixed
+- Minimizing and closing invisible with popup
```

Comment:
"Minimizing" and "closing" are actions, while from my understanding the issue is with UI elements. With that in mind, this sentence can be rephrased to describe the fix in a better way.

Reply:
Done